mllib/src/main/scala/org/apache/spark/ml/r/RWrapperUtils.scala

```diff
@@ -41,5 +41,29 @@ object RWrapperUtils extends Logging {
         s"using new name $newFeaturesName instead")
       rFormula.setFeaturesCol(newFeaturesName)
     }
+
+    if (rFormula.getForceIndexLabel && data.schema.fieldNames.contains(rFormula.getLabelCol)) {
+      val newLabelName = s"${Identifiable.randomUID(rFormula.getLabelCol)}"
+      logWarning(s"data containing ${rFormula.getLabelCol} column and we force to index label, " +
```

Comment:
one concern is this might happen a lot - esp. with libsvm data or similar. Also this is benign correct? Should we just make this INFO so this won't pop up as much?